app/src/main/java/com/hunguet123/searchImage/di/ViewModelModule.kt

```diff
@@ -0,0 +1,11 @@
+package com.hunguet123.searchImage.di
+
+import com.hunguet123.searchImage.ui.MainViewModel
+import org.koin.androidx.viewmodel.dsl.viewModel
+import org.koin.dsl.module
+
+var viewModelModule = module {
```

Comment:
luôn ưu tiên val hơn var
hạn chế dùng var nếu không có nhu cầu chỉnh sửa biến 
```suggestion
val viewModelModule = module {
```